modifiers/modifiers.go

```diff
@@ -16,6 +16,7 @@ func New() *mold.Transformer {
 	mod.Register("lcase", ToLower)
 	mod.Register("ucase", ToUpper)
 	mod.Register("snake", SnakeCase)
+	mod.Register("ucfirst", UCFirstCase)
```

Comment:
since the UCFirstCase function will be exposed can we be more explicit with its naming eg. `UppercaseFirstCharacter`

`ucfirst` is fine for the tag because we want to keep it short and sweet :)

Reply:
I agree. Done.